Adding AppDome documentation (#1619)
saucelabs/sauce-docs: docs/basics/integrations/appdome.md

* A list of your apps will be displayed. If you haven’t uploaded an app before then the list will be empty. 
If your test app does not appear in the above list, you can upload it by clicking **App Upload**.

* After you uploaded your test app, hover your mouse/cursor over the app to display the **Settings** option. Click on Settings.

<img src={useBaseUrl('img/integrations/appdome/appdome-2.png')} alt=""/>

* Disable **Instrumentation** and **Image Injection**. Then click **Back to App Selection**.

<img src={useBaseUrl('img/integrations/appdome/appdome-3.png')} alt=""/>

* Hover your mouse/cursor over the app.  It will show the **Choose Device** option. Click it in order to select the test device.
* Select a test device from the displayed list and click **Launch**. This will start a manual test on the uploaded app on the selected device.

<img src={useBaseUrl('img/integrations/appdome/appdome-4.png')} alt=""/>

You can see live device logs by clicking the **LOG** button on the menu on the right

<img src={useBaseUrl('img/integrations/appdome/appdome-5.png')} alt=""/>

When Done click **STOP** on the right menu

> If you want to test using Appium, check out the [SauceLabs Appium Wiki](https://wiki.saucelabs.com/display/DOCS/Getting+Started+with+Appium+for+Mobile+Application+Testing)

Reviewer: ```suggestion
If you want to test using Appium, see [Appium on Sauce Labs](/mobile-apps/automated-testing/appium/).
```

---
id: appdome
title: Appdome Integration
sidebar_label: Appdome

> description: Test Appdome-secured Android and iOS Apps Using SauceLabs automation test platform for DevSecOps.

Reviewer: ```suggestion
description: Test Appdome-secured Android and iOS apps with Sauce Labs.
```
